.devcontainer/devcontainer.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "Docusaurus Dev Container",
-  "image": "mcr.microsoft.com/vscode/devcontainers/typescript-node:0-10-buster",
+  "image": "mcr.microsoft.com/vscode/devcontainers/typescript-node:12-buster",
   "settings": {
     "terminal.integrated.shell.linux": "/bin/bash"
   },
-  "extensions": ["dbaeumer.vscode-eslint"],
+  "extensions": ["dbaeumer.vscode-eslint", "orta.vscode-jest"],
```

Comment:
I don't remember what this is about, but I trust you for this to be a good change :D

Can we use Node 14 instead?

Reply:
upgraded to node 14!  

The extension is https://marketplace.visualstudio.com/items?itemName=Orta.vscode-jest

Makes it much easier to work with Jest in VS Code - you don't have to take this but I think it's very useful!

![image](https://user-images.githubusercontent.com/1010525/115536121-55e59900-a291-11eb-90f9-68d1552f11d8.png)

Reply:
thanks, let's add this :)